src/transformStateWithClones.js

```diff
@@ -8,6 +8,37 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const stateHistory = [];
+  const stateClone = {};
```

Comment:
The `stateClone` object is initialized as an empty object here. However, it should be a clone of the initial `state` object. Consider using `Object.assign({}, state)` or the spread operator `{...state}` to initialize `stateClone` with the properties of `state`.